### PR TITLE
Prac 08 feedback

### PR DESCRIPTION
**Pull request by Lijiayi2**
@a-rosa
@nora-zyy
@gzy20020626
@AdriantoHartono

### CODE REVIEW

File: prac_08/box_layout_demo.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        self.title = "Box Layout Demo"
+        self.root = Builder.load_file('box_layout.kv')
+        return self.root
+
+    def handle_greet(self):
+        print("test")
+        self.root.ids.output_label.text = f"Hello {self.root.ids.input_name.text}"
+
+    def clear_input(self):
+        self.root.ids.input_name.text = f""
```
Comment (a-rosa) [inline prac_08/box_layout_demo.py:15]: No need the f in front of the  ""

File: prac_08/convert_miles_km.kv [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+            size_hint_x: 0.25
+            Button:
+                text: 'Up'
+                on_press: app.handle_increment(1)
+            Button:
+                text: 'Down'
+                on_press: app.handle_increment(-1)
+#    Button:
+#        text: 'Convert m to km'
+#        on_press: app.handle_calculate()
```
Comment (a-rosa) [inline prac_08/convert_miles_km.kv:21]: I think you are not supposed to make the button section a comment

File: prac_08/dynamic_labels.kv [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+    BoxLayout:
+        orientation: 'horizontal'
+        size_hint_y: 0.2
+        Button:
+            text: "Create Labels"
+            on_press: app.create_labels()
+        Button:
+            text: "Clear Labels"
+            on_press: app.clear_all_labels()
```
Comment (a-rosa) [inline prac_08/dynamic_labels.kv:14]: I think in this program, we are supposed to not have any buttons so, the button section is changed to label and there is not supposed to have any user interaction or input

File: prac_08/dynamic_labels.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+    def create_labels(self):
+        """Create labels from data and add them to the GUI."""
+        for name in self.name_list:
+            label = Label(text=name)
+            self.root.ids.main_layout.add_widget(label)
+
+    def clear_all_labels(self):
+        """Clear all labels"""
+        self.root.ids.main_layout.clear_widgets()
```
Comment (a-rosa) [inline prac_08/dynamic_labels.py:26]: There shouldn't be any clear labels, just create

**Review by AdriantoHartono — COMMENTED** [review]
Nice code, even though there is a little problem. So far I like the code that you have made🙏

File: prac_08/box_layout_demo.py (hunk 1 above)
Comment (nora-zyy) [reply prac_08/box_layout_demo.py:15]: Concise and clear, easy to understand.

**Pull request merged**